### ISSUE TITLE
Model .fits files corrupt in 0.17.4.

### ISSUE THREAD

**Issue by L61** [fermiPy/fermipy#292]
I get a 'File not in FITS or Root format' error when trying to setup a GTAnalysis. I'm finding that a lot of the .fits model files are corrupt (size 1 KB, unopenable) in release 0.17.4. I have attempted to do a clean install but it hasn't worked so far, and only downgrading to 0.17.3 appears to provide working models. This sucks because 0.17.3 is the release with the 'File '[...]/ccube.fits' already exists' bug when loading a previous fit.

This issue is an update on #291, which I closed because it was not representative of the actual problem.

**Comment by TomCayf** [comment]
I'm having the same problem - I'm new to Fermipy and am trying to follow along with the IC443 Jupyter notebook tutorial. It keep saying IC443.fits is "RuntimeError: File not in FITS or Root format". Not sure what to do about this - did you ever find a solution? I'm also not sure how to check which version of Fermipy I'm using or how to downgrade etc...
I'm using (as far as I'm aware) the latest versions for MacOS v10.14

If you found a solution I would much appreciate some help!

**Comment by L61** [comment]
I found the easiest workaround was to downgrade to .3. Then you get 'File '[...]/ccube.fits' every time you load a model, but you can just go "!rm ccube.fits" (costs ~10 secs of compute every time tops) before that and then the program "works".
The other problem is that 4FGL doesn't work with .3 AFAIK, but oh well.
Do update me if you get a better answer.

**Comment by jeget** [comment]
I encountered this problem when gtsrcmaps tried to model an extended source. I got 

> RuntimeError: File not in FITS or Root format: /home/bulat/anaconda3/envs/fermi/lib/python2.7/site-packages/fermipy/data/catalogs/Extended_8years/Templates/S147.fits

FITS files in the Templates directory are 130B in size. The same files on [GitHub](https://github.com/fermiPy/fermipy/tree/master/fermipy/data/catalogs/Extended_8years/Templates) are 40kB. Interestingly, when I tried to download the whole directory with [DownGit](https://minhaskamal.github.io/DownGit/#/home) or [GitZip](http://kinolien.github.io/gitzip/), I got it corrupted (130B files again)! But if I download the single problematic file S147.fits then it's fine. I put this file to the data/catalogs/Extended_8years/Templates directory and the error seems to disappear, so far.
So, a workaround may be just find the not-FITS-or-Root-format file on GitHub and put it to the corresponding place in the fermipy installation.
